internal/utils/utils.go

```diff
@@ -0,0 +1,44 @@
+package utils
+
+import (
+	"regexp"
+	"strings"
+)
+
+func SanitizeDomain(name, suffix string) string {
```

Comment:
Could you please add unit tests for the different variants, to limit the impact don't the code which uses this function?

Reply:
Sure! I'll prepare them this weekend.

Reply:
Done!